app/_hub/kong-inc/key-auth/_index.md

```diff
@@ -384,6 +392,19 @@ Response:
 [consumer-object]: /gateway/latest/admin-api/#consumer-object
 [acl-associating]: /plugins/acl/#associating-consumers
 
+
+### Behavior matrix
```

Comment:
@jschmid1 You mentioned wanting all plugins to have this. Did you mean just a behavior matrix in general or did you want us to pretty much copy/paste this exact table into each plugin doc?

Reply:
just a behavior matrix in general. this needs to figured out for every plugin individually.